### DISCUSSION TITLE
help.led ws2812b not working

### DISCUSSION THREAD

**Discussion by ecubas84** [awawa-dev/HyperHDR#175] [Q&A]
Good I have tried to update my ambilicht and buy a led strip ws2812b, I am unable to make it work, I have configured hyperhdr I have put a signal converter txs0108e but it does not even turn on, the led strip works I have been able to test it but when I connect it the raspberry pi does nothing,
They advise me to give up and buy other LEDs ? which one do you advise, thank you

**Comment by awawa-dev** [comment]
Hi
If you managed somehow in some way to test LED strip then at least from your point of view can be assured that the LED strip is working. Using non-spi LED strip directly from Rpi is not officially supported and you should use ESP with direct connection USB or SPI instead (also using WLED could help the testing even if don't have a plan for using it later) but have few advice:
- txs0108e is as good as using a placebo or worse. It's too slow and many people have a problem with it. OK, it can translate 3.3V to 5V but by doing that it can disrupt the signal. SN74AHCT125N should be use or "sacrifice LED" lever shifter at least.
- if you are sure that you connected everything properly then you can check HyperHDR logs and search for errors related to ws2812b.

**Comment by ecubas84** [comment]
![Captura de pantalla 2021-12-16 151102](https://user-images.githubusercontent.com/96242025/146387836-78480817-73ed-4291-86d6-ac472bec629f.png)
![Captura de pantalla 2021-12-16 151219](https://user-images.githubusercontent.com/96242025/146387885-c11475bf-41be-40ed-98d2-be65124e4861.png)

**Comment by awawa-dev** [comment]
![obraz](https://user-images.githubusercontent.com/69086569/146388749-7e5f22b1-2a36-4f33-bf11-d2f2d0ce1115.png)

**Comment by ecubas84** [comment]
ok thanks i will investigate
